kernelci/cli/__init__.py

```diff
@@ -129,3 +132,48 @@ def group(self, *args, cls=None, **kwargs):
 def kci(ctx, settings):
     """Entry point for the kci command line tool"""
     ctx.obj = CommandSettings(settings)
+
+
+def split_attributes(attributes: typing.List[str]):
+    """Split attributes into a dictionary.
+
+    Split the attributes string into a dictionary using space as a delimiter
```

Comment:
Actually, I just realised there was a problem with the docstring.  I'll send a fixup.  The attributes argument is not a string with spaces but a list of strings already split.